src/discussions/posts/data/thunks.js

```diff
@@ -223,7 +224,7 @@ export function createNewThread({
         following,
         anonymous,
         anonymousToPeers,
-      });
+      }, enableInContextSidebar);
```

Comment:
DiscussionContext can be directly imported to use it here, it would be a cleaner approach, and will not require changes in additional files. i.e `src/discussions/post-comments/comments/comment/CommentEditor.jsx`


```js
import { useContext } from 'react';
import { DiscussionContext } from '../../common/context';


const { enableInContextSidebar } = useContext(DiscussionContext);

Reply:
@AhtishamShahid hooks can only be used in the components. Unfortunately, the above approach won't work here.